plugins/kinet/KiNetNode.h

```diff
@@ -56,6 +57,7 @@ class KiNetNode {
 
  private:
     bool m_running;
+    static ola::SequenceNumber<uint32_t> m_transaction_number;
```

Comment:
Unless your static is upsetting that? Although arguably two nodes would be the same as two copies of Quick Play Pro, which would presumably produce two sets of transaction numbers?

Reply:
Yes, the static was why it needed to be defined separately.

No, when I had two power supplies configured at the same time in one QuickPlay Pro 2 instance it incremented the packet counter globally across all of them. So similarly every packet sent by OLA should be incremented.

Reply:
> No, when I had two power supplies configured at the same time in one QuickPlay Pro 2 instance it incremented the packet counter globally across all of them. So similarly every packet sent by OLA should be incremented.

Yeah but I was asking about two copies of QPP2. The KiNetNode is defined in the plugin so should only exist once (although admittedly I haven't tested that yet...).

Reply:
> Yeah but I was asking about two copies of QPP2. The KiNetNode is defined in the plugin so should only exist once (although admittedly I haven't tested that yet...).

Ah, so there is only one instance of the `KiNetNode` class? In that case this does not need to be static.

Reply:
Well, at least in the tests it is spawned twice, but that is not a big deal. In the plugin it does appear to only be spawned once. It looks like if `StartHook()` is called more than once then it might spawn a second, but I think that's a good opportunity for the counter to reset anyway. I'll fix it.

Reply:
Fixed!